extension/content.js

```diff
@@ -194,6 +215,8 @@ document.addEventListener('DOMContentLoaded', () => {
 	}
 
 	if (isRepo) {
+		const isRepoTree = window.location.href.split('/').length === 5 || window.location.href.split('/')[5] === 'tree';
```

Comment:
removing the test for `window.location.href.split('/')[5] === 'tree'` means you can't edit in non-default branches. E.g. you can't edit stuff in the `development` branch. I think we should leave it, no?